src/commands/run-tests.yml

```diff
@@ -17,6 +17,10 @@ parameters:
     type: boolean
     description: Fail if tests were not triggered or results could not be fetched.
     default: false
+  fail_on_missing_tests:
+    type: boolean
+    description: Fail if at least one test is missing in a run (for example, if it has been removed or deleted).
```

Comment:
```suggestion
    description: Fail if at least one specified test is missing in a run (for example, if it has been deleted).
```

Reply:
I wanted 2 things:
- To make the description shorter in this `run-tests.yml` file, just to be concise, and have the full description in the README, but I agree, let's not do that.
- To avoid using links (especially with markdown syntax), because markdown syntax is [not supported in CircleCI's UI](https://ideas.circleci.com/orbs/p/orb-descriptions-as-markdown) (cf. screenshot)

The backticks in the description of `site` are not rendered as inline code:

![image](https://github.com/DataDog/synthetics-test-automation-circleci-orb/assets/9317502/220a3c76-32c1-4d79-89b7-431c9169e28b)

So @alai97, I'll update this with the text you suggested, but without the hyperlink.

Reply:
Actually, we should not mention ``a `--public-id` CLI argument`` at all in the CI integrations because they don't use the CLI interface of datadog-ci. We should say that they can either use **the `public_ids` parameter** or use a **test file**.

Reply:
For the azure devops extension (cf. [changes](https://github.com/DataDog/datadog-ci-azure-devops/pull/66/files#diff-b01a64d5714cd33f49bd61fd0e113bac3ffee8fda51b22cff49067b1db40e22dR132)), the description **does support Markdown**, so we can have the hyperlink.

I'll also have to rephrase ``a `--public-id` CLI argument`` in that other PR.